### ISSUE TITLE
About TrainingSet and TestSet

### ISSUE THREAD

**Issue by loretoparisi** [NaturalNode/natural#310]
Hi, I have a question about training and test set for the `BayesClassifier`.
I did a simple training with 10.000 text documents in two classes (1 | 0). This will be my `training set`. After the training - i.e. using `classifier.addDocument`, I have saved the classifier, loaded again, and run against the same dataset, but this time using `classifier.classify` api (so being my `test set` here). I can see then that the error rate against the test set seems to be too high for what I would expect, that is more than `0.4` on this dataset. I would expect something like between `0.1-0.2` for the test set.
